components/ViewHabits.tsx

```diff
@@ -1,12 +1,34 @@
 import { css } from '@emotion/react';
 import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import Habit from './Habit';
+import { Habit as HabitType } from '../data-types';
 
-type Props = {};
+const fetchData = async () => {
+  const data = await fetch('../data/habits.json');
+  const json = await data.json();
+  return json;
+};
```

Comment:
nice - it might be good to put this in a separate `data` or `api` folder, so that the implementation of these things can be kept away and together

Reply:
[non-blocking] Sometimes, using promises over async/await can be a little more terse - this is a good example but totally up to you

```suggestion
const fetchData = () => fetch('../data/habits.json').then(res => res.json())
```